src/components/Redemptions/RedemptionsMenu.tsx

```diff
@@ -191,16 +191,16 @@ const SubscribedMenuItemContent = withSubscription({
   loadingComponent: <div>Loading...</div>,
   errorComponent: (props) => <div>{ props.error.message }</div>,
   checkForUpdate: [], // Parent component will rerender anyway.
-  createObservable: (props: IMenuItemProps) => {
+  createObservable: async (props: IMenuItemProps) => {
     const { currentAccountAddress, proposal } = props;
     const arc = getArc();
-    const dao = arc.dao(proposal.dao.id);
-    const ethBalance = concat(of(new BN("0")), dao.ethBalance()).pipe(ethErrorHandler());
-    const rewards = proposal.proposal.rewards({ where: { beneficiary: currentAccountAddress }})
+    const dao = arc.dao(proposal.coreState.dao.id);
+    const ethBalance = concat(of(new BN("0")), await dao.ethBalance()).pipe(ethErrorHandler());
+    const rewards = proposal.rewards({ where: { beneficiary: currentAccountAddress }})
       .pipe(map((rewards: Reward[]): Reward => rewards.length === 1 && rewards[0] || null))
       .pipe(mergeMap(((reward: Reward): Observable<IRewardState> => reward ? reward.state() : of(null))));
     // subscribe to dao to get DAO reputation supply updates
-    return combineLatest(dao.state( { subscribe: true }), ethBalance, rewards);
```

Comment:
why is the subscription removed?

Reply:
whoops, nice catch. Sorry about that, removed it when I was trying to debug the "infinite loading" bug caused by npm linking arc.js.